physionet-django/project/templates/project/project_preview.html

```diff
@@ -221,6 +221,11 @@ <h5 class="card-header">Corresponding Author</h5>
     <!-- /.sidebar -->
   </div>
   <h2>Files</h2>
+  {% if project.embargo_files_days %}
+    <div class="alert alert-warning col-md-8" role="alert">
+      The files in this project version are under embargo until {{ project.embargo_files_days }} days after publication. The files shown below will be hidden in the published project as part of this embargo.
```

Comment:
nitpick but you can use the `|pluralize` tag here for day vs days. I think the syntax is:

```
day{{ project.embargo_files_days|pluralize }} after publication
```

Reply:
I see, thanks for the tip! I can implement this.

Reply:
This has been implemented in the templates.